backend/oasst_backend/prompt_repository.py

```diff
@@ -547,7 +549,7 @@ def store_text_labels(self, text_labels: protocol_schema.TextLabels) -> tuple[Te
                         message_id, protocol_schema.EmojiOp.add, protocol_schema.EmojiCode.red_flag
                     )
 
-                    discord.send_new_report_message(message=message, label_text=text_labels.text, user_id=self.user_id)
+                    send_new_report_message.delay(message=message, label_text=text_labels.text, user_id=self.user_id)
```

Comment:
So I think what we want to do here is add this to a queue (maybe in the backend Redis) and then run the Celery task on a timer, which will send all recently flagged messages as a single Discord message, this way we can avoid spamming Discord with API calls on every flag

Reply:
I see this one, so you mean making this a scheduled task? 

Do you mean having a schedule when to run them? if that is the case do you have an ideas about when we should run them?

Reply:
Yes. I think a fairly short schedule (e.g. once every 10 minutes) would be fine, we just want to avoid constant spam of Discord messages

Reply:
I kinda see, what you mean.

We just need to push those messages into the queue, and then at a given time, fetch the flagged message and send them in Discord.